code/game/objects/items/weapons/clown_items.dm

```diff
@@ -39,6 +39,13 @@
 	//So this is a workaround. This also makes more sense from an IC standpoint. ~Carn
 	if(user.client && (target in user.client.screen))
 		to_chat(user, "<span class='notice'>You need to take that [target.name] off before cleaning it.</span>")
+	else if(target == user && user.a_intent == "grab")
+		var/mob/living/carbon/human/muncher = user
+		var/species = muncher.get_species()
+		if(species == "Drask")
+			to_chat(user, "You take a bite of the [src.name]. Delicious!")
+			playsound(user.loc, 'sound/items/eatfood.ogg', 50, 0)
+			user.nutrition += 5

```

Comment:
The question is whether it's worse to be able to eat soap indefinitely like clowns/mimes and their crayons, or if it's worse to be able to destroy soap (which is fairly limited if chem is absent/busy with more important things)